tests/test_sanitze.py

```diff
@@ -18,11 +16,10 @@
 
 
 @pytest.mark.parametrize(
-    "inp", INVALID_TEST_CASE_ARGS, ids=INVALID_TEST_CASE_IDS
+    "text", INVALID_TEST_CASE_ARGS, ids=INVALID_TEST_CASE_IDS
 )
-def test_sanitize_invalid(inp: str):
-    with pytest.raises(plug.PlugError):
-        _sanitize.sanitize_text(inp)
+def test_sanitize_invalid(text: [str]):
```

Comment:
You have a bad habit of putting type hints in brackets :)
```suggestion
def test_sanitize_invalid(text: str):
```